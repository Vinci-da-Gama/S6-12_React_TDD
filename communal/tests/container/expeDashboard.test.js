import React from 'react';
import { shallow } from 'enzyme';

import LivingExpenseDashboard from '../../src/containers/pages/expeDashboard';

test('should render dashboard correctly.', () => {
    const wrapper = shallow(<LivingExpenseDashboard />);
    expect(wrapper).toMatchSnapshot();
});
