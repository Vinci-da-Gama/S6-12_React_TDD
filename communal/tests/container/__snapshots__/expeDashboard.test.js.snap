// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correctly. 1`] = `
<div
  className="container-fluid"
>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseListCompo) />
  <div
    className="clearfix"
  />
  <div
    className="row mx-3"
  >
    <p
      className="col-12 col-sm-4 col-md-3"
    >
      haha-name
    </p>
    <p
      className="col-12 col-sm-4 col-md-3"
    >
      Age: 
      34
    </p>
    <div
      className="col-12 col-sm-4 col-md-3"
    >
      Job is: 
      Full-Stack_Developer
    </div>
  </div>
</div>
`;
